### ISSUE TITLE
When entity is selected show "last modified" in left pane

### ISSUE THREAD

**Issue by macias** [openstreetmap/iD#9691]
### Description

Let's say I am about to change some road -- add surface information -- I click on the road, in the left pane I see all existing features for it. The one information that would be handy is "Last modified: 2020-07-12" (example).

Rationale: to avoid ping-pong effect I generally stop making changes when the last modification is fresh (like somebody put grade 2, and I want to change it to grade 4). But if the change is old (7 years ago) it indicates that road simply changed in time. Such information does not take too much screen space and it would be valuable.

Currently I have to switch to OSM inspector, which is time consuming (when making a lot of changes).

Extra feature would be showing tooltip with commit comment when moving mouse over "Last modified ..." block.

**Comment by 1ec5** [comment]
There is a History panel (Ctrl+Shift+H, or Shift+Option+H on macOS) that shows the last modification date and user, but it could be made more discoverable.

Note that this information pertains to the feature as a whole, not an individual field. For that, there would need to be integration with something like [osm-blame](https://github.com/raspbeguy/osm-blame/) or an API equivalent to openstreetmap/openstreetmap-website#3535.

**Comment by macias** [comment]
This is very helpful, thank you.

> Note that this information pertains to the feature as a whole, not an individual field. 

Yes, this is fine (as this report goes). By definition if I see the entity was edited 6 years ago it means, every feature/field change is 6 years old or more.

> it could be made more discoverable.

Yes :-) Maybe rewording the history pane would help, because then linking would make sense.

Currently: Edits by A, B, C and 10 more.

Proposed: Last edit YYYY-MM-DD by A. See more...

Benefits:
* this would give you information about date right away, no clicking
* "See more..." would be link to the history, so it would be more discoverable than today
* pretty cheap change

**Comment by nickrsan** [comment]
#7629 is an open pull request to do exactly that. It looks stale - would you have any interest in trying to update it and move it forward? Much of the work might already be complete. We'd also want to check in with the maintainers to confirm they'd be interested in merging the change if it was updated.

**Comment by macias** [comment]
@nickrsan I am sorry but I won't be able to do it. Time + familiarity with JS. Also the history of the other PR does not look promising.
